.buildkite/scripts/steps/integration_tests_tf.ps1

```diff
@@ -0,0 +1,36 @@
+param (
+    [string]$GROUP_NAME,
+    [string]$TEST_SUDO
+)
+
+Write-Output "~~~ Preparing environment"
+
+$PSVersionTable.PSVersion
+
+. "$PWD\.buildkite\scripts\steps\ess.ps1"
+
+# Read package version from .package-version file
+$PACKAGE_VERSION = Get-Content .package-version -ErrorAction SilentlyContinue
+if ($PACKAGE_VERSION) {
+    $PACKAGE_VERSION = "${PACKAGE_VERSION}-SNAPSHOT"
+}
+
+Write-Output "~~~ Building test binaries"
+& mage build:testBinaries
+if ($LASTEXITCODE -ne 0) {
+    Write-Error "Failed to build test binaries"
+    exit 1
+}
+
+$TestsExitCode = 0
+try {
+    # Get-Ess-Stack will start the ESS stack if it is a BK retry, otherwise it will retrieve ESS stack metadata
+    Get-Ess-Stack -StackVersion $PACKAGE_VERSION
+    & "$PWD\.buildkite\scripts\buildkite-integration-tests.ps1" $GROUP_NAME $TEST_SUDO
+    $TestsExitCode = $LASTEXITCODE
```

Comment:
same comment as for the previous ps1 script -- if e.g. both a test fails and ess_down too, the latter doesn't fail the build but it'll be expanded masking the real test issue.